### DISCUSSION TITLE
v7 missing?

### DISCUSSION THREAD

**Discussion by bbredewold** [spatie/laravel-medialibrary#1916] [Q&A] [answered]
Good morning, it seems that the v7 documentation is missing. Anybody knows where it is?

**Comment by miclf** [comment]
[Here?](https://docs.spatie.be/laravel-medialibrary/v7/introduction/)

You can choose use the dropdown at the top of the page to select the version of the documentation that you need :)

**Comment by freekmurze** [comment, marked as answer]
Meanwhile, the problem has been fixed.
